include/deal.II/grid/grid_tools.h

```diff
@@ -1396,7 +1396,7 @@ namespace GridTools
                 // this line has children
                 cell->face(face)->child(0)->vertex(1)
                   = (cell->face(face)->vertex(0) +
-                     cell->face(face)->vertex(1)) / 2;
+                     cell->face(face)->vertex(1)) / 2.;
```

Comment:
Uh, do you need this now? Surely I would expect that you can write `Point<dim>()/2` without a compiler error...